src/scripts/const/arbitrum.ts

```diff
@@ -0,0 +1,41 @@
+export const ARBITRUM_BRIDGE_ADDRESS = '0x09e9222e96e7b4ae2a407b98d48e330053351eee'
+
+/**
+ * Most of the tokens that are bridged to Arbitrum using the bridge, they don't have liquidity, so the trade experience is not good.
+ * This list is meant to limit the tokens shown to the ones that have liquidity.
+ * 
+ * Future versions of the script should decide using objective on-chain data if a token is liquid or not.
+ */
+export const TOKENS_WITH_LIQUIDITY = [
+  'ARB', 'cbETH', 'GRT', 'LINK', 'USDC', 'USDT', 'WBTC', 'WETH',
```

Comment:
Does it mean that we are going to have only 8 tradable tokens on Arbitrum?
Looks a bit poor

Reply:
We have liquidity for COW but it's not in the list

Reply:
no no, this is a placeholder for tomorrow. 

Chen is working on the actual list. There will be many more